mmdet/models/utils/misc.py

```diff
@@ -1,7 +1,34 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from torch.autograd import Function
 from torch.nn import functional as F
 
 
+class SigmoidGeometricMean(Function):
```

Comment:
How about we implement an interface named sigmoid_geometric_mean = SigmoidGeometricMean.apply here so that in tood_head we can simply use sigmoid_geometric_mean(xxx)?

Reply:
We have to use `SigmoidGeometricMean.apply` in the high PyTorch version such as  Pytorch 1.10